main/store/actions/index.js

```diff
@@ -496,10 +503,20 @@ module.exports = {
     })
   },
   setBlockHeight: (u, chainId, blockHeight) => {
-    u('main.networksMeta.ethereum', chainId, (chainMeta) => ({ ...chainMeta, blockHeight }))
+    u('main.networksMeta.ethereum', chainsMeta => {
+      if (chainsMeta[chainId]) {
```

Comment:
when can this not be defined?

Reply:
If an invalid `chainId` is passed to the action it can put Frame in an unrecoverable state

Reply:
don't love this type of defensive code because now this will be a no-op that can cover up something else that's wrong. at the very least we should log an error here

Reply:
or report one to Sentry so we can fix it